apps/activitypub/src/views/profile/profile.tsx

```
@@ -1,8 +1,8 @@
-import ActorList from './components/ActorList';
-import Error from '@components/layout/Error';
-import Likes from './components/Likes';
-import Posts from './components/Posts';
-import ProfilePage from './components/ProfilePage';
+import ActorList from './components/actor-list';
+import Error from '@components/layout/error';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)